### DISCUSSION TITLE
Prevent "Reused translation" check when Singular and Plural the same in the translated language

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#10369] [Q&A]
The example is about the English terms `Error` and `Errors`. In German they are translated both as `Fehler`. There is no difference between Singular and Plural for this term in German.

I do understand why Weblate do report a "Reused translation" check error at this point.
![image](https://github.com/WeblateOrg/weblate/assets/11861496/4d4922bf-1cd3-4d5e-a498-5b3cb863f6bb)

Is there a way to prevent such false positive checks?

I do know about GNU gettext plural terms feature. But I'm not sure if this is the right answer. Are there other ways?

In Python it would result in a lot of lines like this:

    ngettext('Error', 'Errors', 1)  # Singular
    ngettext('Error', 'Errors', 2)  # Plural

See similar discussion at #10370

**Comment by buhtz** [comment]
I would be glad about a solution different from `ngettext()`. As you can see in [my PR](https://github.com/bit-team/backintime/pull/1560) and [this discussion on reddit](https://www.reddit.com/r/translator/comments/17r9is8/english_arabic_polish_correct_plural_form_of) it is not easy to decide about the correct integer used for the plural form in the `ngettext()` function.

Languages like Arabic or Polish do have multiple plural forms depending on the number/count used.

**Reply by nijel** [reply]
Yes, that's why you should use `ngettext()` and Gettext logic will find the correct plural. There is no need for separate plural string then.

**Reply by buhtz** [reply]
But it is a plural without a specific count. That is the point here.
